### ISSUE TITLE
Cannot install plugins

### ISSUE THREAD

**Issue by filviu** [MrCarroll/joplin-desktop-snap#26]
Hi. I'm a bit new to snaps. I'm running the latest stable snap `v2.12.18` and if I try to install any plugin I get the following:

![image](https://github.com/MrCarroll/joplin-desktop-snap/assets/2470335/c774f974-d07a-4767-9bfa-c75e51ab613e)

I tried multiple plugins all with the same error. 

Sorry I tried but couldn't find a way to copy paste from the error popup...

Stock Ubuntu 23.04 all updates applied as of today. Anything I'm doing wrong or missed doing ? Thanks!

**Comment by JGCarroll** [comment]
Hi,

I've just tried installing a few plugins now on my end and not had an issue. Could you try running `sudo snap refresh`, primarily to ensure the base snap (Core22) and content snaps (E.G Gnome) are all updated.

It might be this is happening for only a certain plugin, if so, could you let me know the name so I can try it on my end?

**Comment by filviu** [comment]
You are right, it's me.

I tested Ubuntu 22.04 and 22.03 in VMs thinking stock 22.03 introduced something that 22.04 didn't trigger(hurray for quickemu) but they work fine. So I went back to my desktop uninstalled and reinstalled joplin and plugins now install correctly.

I must have messed something up when I setup my machine ( moved home to an encrypted partition - maybe then I borked something) 

Sorry to have wasted your time.
